assets/admin/exit-survey/form.js

```diff
@@ -0,0 +1,67 @@
+import { useCallback, useRef, useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { ExitSurveyFormItem } from './form-item';
+import { reasons } from './reasons';
+
+/**
+ * Exit survey form.
+ *
+ * @param {Object}   props
+ * @param {Function} props.submit Callback to submit feedback.
+ * @param {Function} props.skip   Callback to skip feedback.
+ */
+export const ExitSurveyForm = ( { submit, skip } ) => {
+	const form = useRef( null );
+	const [ , updateInput ] = useState( null );
```

Comment:
Is there a purpose to this state (maybe to force a re-render in some case?)? Or did we just missed that in a change?

Reply:
Yes this is for rerendering — the form inputs are uncontrolled (which avoids some complexity and extra code), but updating on change is still needed to toggle the Submit button disabled state.